tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -143,6 +144,117 @@ def test_fail_on_multiple_drivers():
         extract_db_type_from_uri("mysql+pymsql+pyodbc://...")
 
 
+@pytest.fixture
+def test_database(tmp_path: Path) -> SqlAlchemyStore:
```

Comment:
```suggestion
def store(tmp_path: Path) -> SqlAlchemyStore:
```

can we rename this fixture?

Reply:
Yes, since the schema get changed to SqlAlchemyStore, let me rename it.